lib/render.module.ts

```diff
@@ -23,6 +23,15 @@ export class RenderModule {
       await next.prepare();
     }
 
+    const nextConfig = (next as any).nextConfig;
+    const nextServer = (next as any).server;
```

Comment:
Depending on Next version (^9 or ^10) `nextConfig` property may be in a different location. In older Next ^9 we have direct access to it with proper typings. In Next ^10 I wasn't able to find any way to access the config from injected Next Server instance, however there is a way to get it from an untyped `server` property. Tested this on REPL repo provided in #79 with Next 10.2 and on an enterprise project with Next 9.5.